app/views/insured/families/_enrollment_status_label.html.erb

```diff
@@ -1,3 +1,3 @@
 <div class="text-right">
-   <%= enrollment_state_label hbx_enrollment %>
+   <%= enrollment_state_label(hbx_enrollment, @bs4.present?) %>
```

Comment:
small thing, I know we can for now guarantee when @bs4 is present, it is also true. but that contract could break in the future, why not `@bs4 == true`?